octobot_trading/util/test_tools/exchanges_test_tools.py

```diff
@@ -245,5 +253,7 @@ async def wait_for_other_status(order: personal_data.Order, timeout) -> personal
             return personal_data.create_order_instance_from_raw(
                 order.trader, raw_order, force_open_or_pending_creation=False
             )
+        if time.time() - t0 + constants.CREATED_ORDER_FORCED_UPDATE_PERIOD >= timeout:
+            break
```

Comment:
should we add a debug log here or it's not necessary?

Reply:
No we don't this check is just to kip an asyncio.sleep when we know the next while condition will be False (and therefore we sleep for nothing)